source/style-guide/quickstart.txt

```diff
@@ -86,12 +86,11 @@ the following ones to help you boost the effectiveness of your writing.
      - :ref:`use-correct-punctuation`
 
        :ref:`punctuation`
-   * - **Use sentence-style capitalization for all titles and
+   * - **Use headline-style capitalization for most titles and
        headings**.
 
-       In sentence-style capitalization, you capitalize only the first
-       word of the title or heading, plus any proper terms and terms
-       that are always capitalized, such as some abbreviations.
+       To meet headline-style capitalization requirements, capitalize the initial 
+       letter of the first, last, and all significant words in a phrase.
```

Comment:
```suggestion
       letter of the first, last, and all words in a phrase except for articles,
       coordinating conjunctions, prepositions, and the types of words outlined in the 
       guidelines.
```

Reply:
I think "significant" is too vague in this context.

Reply:
agreed, will make this change!